src/contents/shared/ProfileEdit.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
```

Comment:
Please wrap the profile edit into the NavFooterWrapper Component. The nav and the footer always have to be visible.

Reply:
Done!